index.html

```diff
@@ -10,60 +10,167 @@
 </head>
 <body>
     <div class="body-wrapper">
-        <nav class="header-flex">
-            <a class="logo" href=""><h5>Omar Mohammad</h5></a><span>
-            <img class="union" src="./assets/Union.png" alt='union-sign'/></span>
-        </nav>
-        <div class="container-flex">
+        <section class="header">
+            <ul class="nav" id="nav">
+                <li><h5 id="logo">Omar Mohammad</h5></li>
+                <li>
+                    <span>
+                        <img id="union" src="./assets/Union.png" alt='union-sign'/>
+                    </span>
+                </li>
+            </ul>
+        </section>
+        <section class="container-flex">
             <h1>Omar Mohammad</h1>
             <br>  
             <h2>Software Developer</h2>        
             <br/>
             <p class="about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <br>
             <hr/>
-        </div>
-        <br/>
-        <br/>
-        <div class="skills-flex">
-            <div class="languages">
-                <h4>Languages</h4>
-                <ul>
-                    <li>JavaScript</li>
-                    <li>Ruby</li>
-                    <li>Css</li>
-                    <li>Html</li>
-                </ul>
-            </div>
-            <div class="frameworks">
-                <h4>Frameworks</h4>
-                <ul>
-                    <li>Bootstrap</li>
-                    <li>Ruby on Rails</li>
-                    <li>RSPec</li>
-                    <li>CapyBara</li>
-                    <li>Selenium</li>
-                </ul>
+            <div class="skills-flex">
+                <div class="languages">
+                    <h4>Languages</h4>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>Css</li>
+                        <li>Html</li>
+                    </ul>
+                </div>
+                <div class="frameworks">
+                    <h4>Frameworks</h4>
+                    <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>                        
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="skills">
+                        <h4>Skills</h4>
+                    <ul>
+                        <li>Git</li>
+                        <li>Codekit</li>
+                        <li>Codepen</li>
+                        <li>Terminal</li>
+                        <li>GitHub</li>
+                    </ul>
+                </div>
```

Comment:
- [OPTIONAL] 🛑 ⚠️  Hi @OmarMWarraich , You did a great job on this project, and my apologies the previous reviewer did not mention it - The projects/works list UI does not match the [Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10). It creates an overflow on the x-axis on mobile view  (See the issue in the image attached and compare it with the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)). Kindly see the project design for reference. 

😃  I'll approve this project because you made an effort to implement the suggestions made by the previous reviewer. 👀  But, I kindly ask that you fix the above-mentioned issue for future reviews. 🙏🏽 

# Issue;

![image](https://user-images.githubusercontent.com/35315311/178084017-77e4a3c2-6e33-4609-883d-6d160f6d5910.png)